cheroot/test/test_conn.py

```diff
@@ -412,7 +406,7 @@ def test_keepalive(test_client, http_server_protocol):
     original_server_protocol = test_client.server_instance.protocol
     test_client.server_instance.protocol = http_server_protocol
 
-    http_client_protocol = 'HTTP/1.0'
+    http.client_protocol = 'HTTP/1.0'
```

Comment:
@jaraco this looks wrong :point_up:

Reply:
I guess the regexp used was missing `\b`...

Reply:
Oh. Good catch. Surprised linters and tests failed to catch that.

Reply:
Oh, after looking more closely, I see why. The variable can exist locally or globally or in another module and it doesn't affect the intention.

Reply:
Rewritten in f2e19b67.